homeassistant/components/webostv/triggers/turn_on.py

```diff
@@ -6,18 +6,22 @@
 from homeassistant.const import ATTR_DEVICE_ID, ATTR_ENTITY_ID, CONF_PLATFORM
 from homeassistant.core import CALLBACK_TYPE, HomeAssistant, callback
 from homeassistant.helpers import config_validation as cv
-from homeassistant.helpers.trigger import TriggerActionType, TriggerInfo
+from homeassistant.helpers.trigger import (
+    PluggableAction,
+    TriggerActionType,
+    TriggerInfo,
+)
 from homeassistant.helpers.typing import ConfigType
 
 from ..const import DOMAIN
+from ..device_trigger import async_get_turn_on_trigger
 from ..helpers import (
-    async_get_client_wrapper_by_device_entry,
     async_get_device_entry_by_device_id,
     async_get_device_id_from_entity_id,
 )
 
 # Platform type should be <DOMAIN>.<SUBMODULE_NAME>
-PLATFORM_TYPE = f"{DOMAIN}.{__name__.rsplit('.', maxsplit=1)[-1]}"
+PLATFORM_TYPE: str = f"{DOMAIN}.{__name__.rsplit('.', maxsplit=1)[-1]}"
```

Comment:
Mypy gives really odd error of
```
homeassistant/components/webostv/device_trigger.py:25: error: Cannot determine type of "PLATFORM_TYPE"  [has-type]
```

My guess would be something along the line that it's using __name__ which might not be fully available when imported from another file. But i can not explain it.